src/Logging.W3C/src/Microsoft.Extensions.Logging.W3C.csproj

```
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Logger implementation to support W3C logging</Description>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <IsShipping>true</IsShipping>
```

nit: We're inconsistent but lean toward

```suggestion
    <IsShippingPackage>true</IsShippingPackage>
```

`$(IsShipping)` controls `$(IsShippingPackage)` and `$(IsShipping)` defaults to `true` for implementation projects. So, this project should default to having both `$(IsShippingPackage)` and `$(IsShipping)` set to `true`. Could just delete the line but being explicit doesn't hurt.

---

https://github.com/dotnet/aspnetcore/pull/33251/commits/fcf767132a91adb16682c7b8afc1f4a628194c23